source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,97 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+You can remove any unused index that isn't a :ref:`unique index
+<unique-indexes>`. 
+
+The following sections show the three ways to remove indexes:
+
+- Using an index specification document
+- Using an indexed name field
+- Removing all indexes
+
+Index Specification Document
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the index specification documenent to the ``dropIndex()`` method to
+remove an index from a collection. 
+
+The following snippet removes an ascending index on the ``title`` field
+in a collection:
+
+.. code-block:: java
+
+   collection.dropIndex(Indexes.ascending("title"));
+
+.. important::
+
+   This does not work for a text index. To remove a text index, see the
+   :ref:`Name Field <name_field>` section.
```

Comment:
"This does not work for..." seems like an announcement of a bug. Instead, consider mentioning that text indexes are dropped using a name field.
```suggestion
   If you want to drop a text index, you must use the name of the index instead. See the 
   :ref:`Name Field <name_field>` section for details.
```